forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-store/forgerock-openbanking-uk-aspsp-rs-mock-store-server/pom.xml

```diff
@@ -117,6 +117,7 @@
         <dependency>
             <groupId>com.forgerock.openbanking.uk</groupId>
             <artifactId>openbanking-uk-datamodel</artifactId>
+            <version>3.1.8.3-SNAPSHOT</version>
```

Comment:
I've changed this in the openbanking-parent and then building ASPSP with the SNAPSHOT of the parent. No change to all the POMs then

Reply:
Or rather just one change in the aspsp root pom to use the SNAPSHOT parent version that in turn uses the SNAPSHOT datamodel version

Reply:
I have deleted the uk-datamodel snapshot version to use the parent snapshot version, but the correct snapshot version was `1.2.7-SNAPSHOT` instead of `1.2.6-SNAPSHOT` because the `release 1.2.6` exist and is ready to use.